CanalTP/NmmPortalBundle
# Synchro with FenrirApi on create/update customer

Needs:
 - [ ] https://github.com/CanalTP/fenrir-api-client/pull/4

This PR create an user on Fenrir when a customer is created, and update the user when the customer is updated.

To test it:
 - update a customer, check if username on fenrir is updated as well (only username is needed for NMM)
 - create an user when a customer is created, check if an user is created with the expected username, an origin `nmm`
